Initial commit: Added Dockerfile and README for Cell Ranger software

jaamarks opened the pull request:
Initial commit for Cell Ranger software tool: https://www.10xgenomics.com/support/software/cell-ranger

We did not include the human genome reference since it would make the docker image over 32GB.

jkstratford reviewed (changes requested):
A few comments, do you typically try and build the docker before approving it? Or do you just trust that the developer has done this prior to submitting the pull request?

jaamarks commented on cellranger/v7.2.0/Dockerfile:
```diff
@@ -0,0 +1,38 @@
+# Set maintainer label
+LABEL maintainer="Jesse Marks <jmarks@rti.org>"
+LABEL description="Cell Ranger software for single-cell analyses."
+LABEL software-website="https://support.10xgenomics.com/single-cell-gene-expression/software/overview/welcome"
```
ACK

jaamarks commented on cellranger/v7.2.0/Dockerfile:
```diff
@@ -0,0 +1,38 @@
+RUN apt-get update && apt-get install -y \
+    curl \
+    vim \
+    && rm -rf /var/lib/apt/lists/*
```
ACK

jaamarks commented:
> A few comments, do you typically try and build the docker before approving it? Or do you just trust that the developer has done this prior to submitting the pull request?

Whoever is reviewing it should try to build the docker image before approving it. This brings up a good point though, maybe we should add a short section on **Reviewing a PR** to our https://github.com/jaamarks/biocloud_docker_tools/tree/master#dockerfile-best-practices-checklist

I'm thinking:
* **Review code**: Check for syntax errors, coding standards, and best practices.
* **Review Documentation**: Ensure that comments and READMEs are clear and informative.
* **Approve**: Verify that the Dockerfile builds locally. Then approve for merging.

jaamarks commented on cellranger/v7.2.0/README.md:
```diff
@@ -0,0 +1,79 @@
+
+Cell Ranger is a set of analysis pipelines that process Chromium Next GEM single-cell data to align reads, generate feature-barcode matrices, perform clustering, and other secondary analyses. It supports various workflows and libraries. For detailed usage instructions and documentation, visit the [official Cell Ranger documentation](https://www.10xgenomics.com/support/software/cell-ranger/getting ...
+
+## Usage
```
ACK

jaamarks commented on cellranger/v7.2.0/README.md:
```diff
@@ -0,0 +1,79 @@
+`docker run -it b55e1a301011 cellranger --help`
+
+Note that we do not include the Human reference (GRCh38) dataset required for Cell Ranger (https://cf.10xgenomics.com/supp/cell-exp/refdata-gex-GRCh38-2020-A.tar.gz)
+Because when these data are extracted the Docker image size becomes over 32GB.
```
ACK

jaamarks commented:
@jkstratford, thanks for the review. I believe I have addressed all of your comments. 
@rmathur87, would appreciate a quick review if you have time also.
